packages/app-root/next.config.mjs

```diff
@@ -11,11 +11,15 @@ const nextConfig = {
       '@zooniverse/user',
       'grommet',
       'grommet-icons'
-    ],
+    ]
   },
   webpack: (config, options) => {
     config.resolve = {
       ...config.resolve,
+      alias: {
+        ...config.resolve.alias,
+        hexoid: 'hexoid/dist/index.js'
+      },
```

Comment:
This punts the issue of upgrading Superagent, but does fix the build errors 🤔

Reply:
I'm really curious as to why the pages router doesn't have a problem with this. We run some stuff on 14.2, and the upgrade from 13.5 was straightforward, but we're using the pages router.